tilavarauspalvelu/admin/email_template/forms.py

```diff
@@ -162,384 +149,432 @@ class PendingInstructionsFormMixin(forms.Form):
-            unit_name=self.cleaned_data["unit_name"],
-            unit_location=self.cleaned_data["unit_location"],
-            begin_datetime=self.cleaned_data["begin_datetime"],
-            end_datetime=self.cleaned_data["end_datetime"],
-            booking_number=self.cleaned_data["booking_number"],
+            reservation_id=self.cleaned_data["reservation_id"],
         )
 
 
-class StaffNotificationReservationRequiresHandlingEmailTemplateTesterForm(
-    ReservationBasicInfoFormMixin,
-    EmailRecipientFormMixin,
-    LanguageFormMixin,
-    BaseEmailTemplateForm,
-):
+class StaffNotificationReservationRequiresHandlingEmailTemplateTesterForm(ReservationBaseForm):
     reservation_name = forms.CharField(initial="[VARAUKSEN NIMI]", widget=text_widget)
     reservee_name = forms.CharField(initial="[VARAAJAN NIMI]", widget=text_widget)
-    booking_number = forms.IntegerField(initial=0, widget=number_widget)
+    reservation_id = forms.IntegerField(initial=0, widget=number_widget)
 
     @classmethod
-    def from_reservation_unit(cls, instance: ReservationUnit, *, language: Lang) -> Self:
-        return cls(
-            initial={
-                "reservation_unit_name": get_attr_by_language(instance, "name", language),
-                "unit_name": get_attr_by_language(instance.unit, "name", language),
-                "unit_location": instance.actions.get_address(),
-            },
-        )
+    def get_initial_data_from_reservation_unit(cls, instance: ReservationUnit, *, language: Lang) -> Self:
+        return {
+            **super().get_initial_data_from_reservation_unit(instance, language=language),
+        }
 
     def to_context(self) -> EmailContext:
         return get_context_for_staff_notification_reservation_requires_handling(
-            language=self.cleaned_data["language"],
+            **super().get_context_params(),
             reservee_name=self.cleaned_data["reservee_name"],
             reservation_name=self.cleaned_data["reservation_name"],
-            reservation_unit_name=self.cleaned_data["reservation_unit_name"],
-            unit_name=self.cleaned_data["unit_name"],
-            unit_location=self.cleaned_data["unit_location"],
-            begin_datetime=self.cleaned_data["begin_datetime"],
-            end_datetime=self.cleaned_data["end_datetime"],
-            booking_number=self.cleaned_data["booking_number"],
+            reservation_id=self.cleaned_data["reservation_id"],
+        )
+
+
+class StaffNotificationApplicationSectionCancelledTemplateTesterForm(EmailRecipientFormMixin, BaseEmailTemplateForm):
+    application_section_name = forms.CharField(initial="[HAKEMUKSEN OSAN NIMI]")
+    application_round_name = forms.CharField(initial="[KAUSIVARAUSKIERROKSEN NIMI]")
+    cancel_reason = forms.CharField(initial="[PERUUTUKSEN SYY]", widget=text_widget)
+
+    def to_context(self) -> EmailContext:
+        return get_context_for_application_section_cancelled(
```

Comment:
Wrong method, should use `get_context_for_staff_notification_application_section_cancelled`.